### DISCUSSION TITLE
Storage Pool 2 can be assembled

### DISCUSSION THREAD

**Discussion by enigmacarpc** [007revad/Synology_HDD_db#122] [Q&A]
Hello all I just added my m.2 drive as a storage pool yesterday but I dint find this script until after I already set everything up. My Synology 420+ shuts down every night at 2:00 AM and when it booted at 9:00 AM it had error beeps. I looked and the drive was removed by Synology, I found this amazing script late yesterday and remembered this script had the fix so I ran syno_hdd_db.sh and it restored the drive and it is fully functional but I have one issue I am getting an alert that Storage Pool 2 can be assembled but when I try to run the drive assembly the option is not there. Any ideas?

**Comment by 007revad** [comment]
Do you have an Available pool showing in the storage manager? See [this screenshot](https://github.com/007revad/Synology_M2_volume/blob/main/images/create_m2_volume_online_assemble.png) If you just click on the **...** on the far right.

Otherwise I'd delete the storage pool and recreate it from storage manager. If storage manager still won't let you create the storage pool then run https://github.com/007revad/Synology_enable_M2_volume, reboot, then create the storage pool.

**Reply by enigmacarpc** [reply]
Hello @007revad Thanks for the fast reply, There are no available pools in the storage manager. Will deleting the storage pool delete my data? Or will running the M2_Volume script destroy my data?

**Reply by 007revad** [reply]
Deleting the storage pool **will** delete your data. Running the Synology_M2_volume script **will** destroy your data.

Running the Synology_**enable**_M2_volume script will **NOT** delete your data. It makes DSM think your drives are Synology drives so it might make DSM show your storage pool in storage manager.

**Comment by enigmacarpc** [comment]
Thanks for the info, I am going to have to leave it or find a way to enable
trim via cli. I have all my dockers on the drive along with hours of
metadata changes.



On Sun, Jul 30, 2023 at 3:51 AM Dave Russell ***@***.***>
wrote:

> Deleting the storage pool *will* delete your data. Running the
> Synology_M2_volume script *will* destroy your data.
>
> Running the Synology_*enable*_M2_volume script will *NOT* delete your
> data. It makes DSM think your drives are Synology drives so it might make
> DSM show your storage pool in storage manager.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/007revad/Synology_HDD_db/discussions/122#discussioncomment-6586171>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AG37HFGZIJA4ZVVDLJVDMCTXSYG7ZANCNFSM6AAAAAA24RUKHU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Tom Vance
732-771-6618
